tdrs-backend/tdpservice/settings/local.py

```diff
@@ -1,13 +1,7 @@
 """Define configuration settings for local environment."""
 import os
-from os.path import dirname, join
-from dotenv import load_dotenv
 from .common import Common
 
-# load env vars from .env file and override any system environment variables
-dotenv_path = join(dirname(__file__), "./env_vars/.env.local")
-load_dotenv(dotenv_path)
```

Comment:
👍 👍 These are some satisfying lines of deleted code! These ENV vars are read in through docker-compose now!

Reply:
@jtwillis92 Am I right in thinking that this PR removes the need for `python-dotenv`? 

Should we remove `python-dotenv` from the Pipfile as an unused dependency?

Reply:
Yes, that is correct - docker-compose will handle all local environment variables now so I'll remove that package.

Reply:
Noting that during pairing today @jtwillis92 ran into some issues removing this package, we could make this one a separate issue if needed.